src/Components/ScrivitoExtensions/ColumnsEditorTab.js

```diff
@@ -244,148 +244,150 @@ const Alignment = Scrivito.connect(({ widget }) => {
   }
 });
 
-class GridLayoutEditor extends React.Component {
-  constructor(props) {
-    super(props);
+const GridLayoutEditor = Scrivito.connect(
```

Comment:
* [ ] FYI This component was nicely separated from Scrivito so far. I vote for for keeping it like that, by adding a `readOnly` prop.